client/lib/desktop/index.js

```diff
@@ -139,6 +142,25 @@ const Desktop = {
 		this.navigate( '/help' );
 	},
 
+	editorLoadedStatus: function() {
+		let previousLoaded = isEditorLoaded( this.store.getState() );
+
+		this.store.subscribe( () => {
+			const loaded = isEditorLoaded( this.store.getState() );
+
+			if ( loaded !== previousLoaded ) {
```

Comment:
If `isEditorLoaded` is already `true` at the moment when `editorLoadedStatus()` is called (not sure if it can ever happen), the `editor-loaded` event will never be fired. Is that ok?

Reply:
Good catch. I think that scenario is unlikely, but I added a safeguard in 3d35238 just to be on the safe side.